packages/iov-socket/src/queueingstreamingsocket.ts

```diff
@@ -8,20 +8,23 @@ import { StreamingSocket } from "./streamingsocket";
  * A wrapper around StreamingSocket that can queue requests.
  */
 export class QueueingStreamingSocket {
-  public readonly connected: Promise<void>;
+  public connected: Promise<void>;
```

Comment:
I don't think this API is very useful, as it only tells you that _some_ socket connected, which says very little about the connection status. I think we can just drop it.

In the one place I found where it is used, we probably don't need it:

```ts
      socket.connect();
      // tslint:disable-next-line:no-floating-promises
      socket.connected.then(() => {
        requests.forEach(request => socket.queueRequest(request));
      });
});
```

-> 

```ts
      socket.connect();
      requests.forEach(request => socket.queueRequest(request));
});
```